browser_history/utils.py

```diff
@@ -178,7 +179,13 @@ def get_browser(browser_name):
         browser_class = None
         for browser in get_browsers():
             if browser.__name__.lower() == browser_name.lower():
-                browser_class = browser
+                support_check = {
+                    Platform.LINUX: browser.linux_path,
+                    Platform.WINDOWS: browser.windows_path,
+                    Platform.MAC: browser.mac_path
+                }
+                if support_check.get(get_platform()) is not None:
+                    browser_class = browser
```

Comment:
Maybe refactor this as a separate internal function? (or a member function of `generic.Browser`?)

Reply:
okay. Let me do that.